src/Symfony/Bridge/Doctrine/ArgumentResolver/EntityValueResolver.php

```diff
@@ -214,11 +218,25 @@ private function getCriteria(Request $request, MapEntity $options, ObjectManager
     private function findViaExpression(ObjectManager $manager, Request $request, MapEntity $options): object|iterable|null
     {
         if (!$this->expressionLanguage) {
-            throw new \LogicException(\sprintf('You cannot use the "%s" if the ExpressionLanguage component is not available. Try running "composer require symfony/expression-language".', __CLASS__));
+            throw new \LogicException(sprintf('You cannot use the "%s" if the ExpressionLanguage component is not available. Try running "composer require symfony/expression-language".', __CLASS__));
+        }
+
+        $variablesFormInjectors = [];
+        foreach($options->modifiersInjectors ?? $this->defaulInjectors as $injector) {
+            $injectorObject = $this->variablesInjectorLocator->get($injector);
+
+            if (!$injectorObject instanceof EntityValueResolverExpressionModifiersInjectorInterface) {
+                throw new \LogicException(sprintf('You cannot use "%s" as Expression Modifier, it must implement "%s" instead.', $injector, EntityValueResolverExpressionModifiersInjectorInterface::class));
```

Comment:
I think the "must" word in messages is avoided in the rest of the codebase if I remember correctly 🤔